### PR TITLE
v1.4.1

### PR DESCRIPTION
**Pull request by alexosugo**

### CODE REVIEW

File: app/src/main/java/com/hover/stax/bounties/BountyAsyncCaller.java [hunk 1]
```diff
@@ -14,18 +13,21 @@
...
 
 import okhttp3.MediaType;
 import okhttp3.OkHttpClient;
 import okhttp3.Request;
 import okhttp3.RequestBody;
 import okhttp3.Response;
+import timber.log.Timber;
 
-class BountyAsyncCaller extends AsyncTask<String, Void, Integer> {
+class BountyAsyncCaller extends AsyncTask<String, Void, Map<Integer, String>> {
```
Comment (davkutalek) [inline app/src/main/java/com/hover/stax/bounties/BountyAsyncCaller.java:26]: This should be replaced with modern implementation

**Review by davkutalek — APPROVED** [review]
I don't see any issues with anything, but given the size of the diff and changes in tabbing as well as java > kt there is plenty that I didn't look at in detail. Is there anything you want me to look at in particular?

File: .github/workflows/prod-release.yml [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+        with:
+          releaseDirectory: app/build/outputs/bundle/release
+          signingKeyBase64: ${{ secrets.SIGNING_KEY }}
+          alias: ${{ secrets.ALIAS }}
+          keyStorePassword: ${{ secrets.KEY_STORE_PASSWORD }}
+          keyPassword: ${{ secrets.KEY_PASSWORD }}
+
+
+      # Create release with tag on Github
+      - name: Create Release
```
[the remark this answers is not in the document]
Comment (alexosugo) [reply .github/workflows/prod-release.yml:76]: I'll need file to edit this file to just create the release tag and not the release build. That way it doesn't break the current release structure we have.

File: app/src/main/java/com/hover/stax/bounties/BountyAsyncCaller.java (hunk 1 above)
Comment (alexosugo) [reply app/src/main/java/com/hover/stax/bounties/BountyAsyncCaller.java:26]: Yes. I'm gonna add this to Pivotal.

**Pull request closed without merging**